prettier/main.ts

```diff
@@ -14,7 +14,7 @@
 const { args, exit, readFile, writeFile } = Deno;
 type FileInfo = Deno.FileInfo;
```

Comment:
This line seems unnecessary now.

Reply:
It looks like it, but actually it's not allowed to destructure the FileInfo interface from he Deno object. I think it's best to keep this.